src/entity/src/user.js

```diff
@@ -889,13 +889,25 @@ export default class User {
   }
 
   /**
+   * Find users.
+   *
+   * @param {Query} [query] Query used to filter entities.
+   * @param {Object} [options] Options
+   * @param {boolean} [options.discover] Discover users.
+   * @return {Observable} Observable.
+   */
+  static find(query, options = {}) {
```

Comment:
The `UserStore` extends the `NetworkStore` which provides some implementation that would otherwise be duplicated in the `User` class. However, I think at this point we should just expose the `User` class that is used to proxy the calls to the `UserStore` class.